packages/table/src/headers/columnHeader.tsx

```diff
@@ -77,7 +77,7 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, {}> {
 
     public render() {
         const { grid, viewportRect, columnIndexStart, columnIndexEnd } = this.props;
-        const cells: React.ReactElement<any>[] = [];
+        const cells: Array<React.ReactElement<any>> = [];
```

Comment:
@adidahiya what's all this `Array<T>` business? what's wrong with good old-fashioned `T[]`?